pkg/executor/nginx/nginx_test.go

```diff
@@ -776,3 +807,28 @@ func cleanString(in string) string {
 	replacer := strings.NewReplacer("\n", "", "\t", "")
 	return replacer.Replace(in)
 }
+
+func validateNginxConfig(t *testing.T, config string) {
+	//Not relevant for syntax checking
+	config = strings.Replace(config, "include       /etc/nginx/mime.types;", "", -1)
```

Comment:
This looks a little precarious; if the number of spaces change, this won't remove the line.

Reply:
Correct, but then the tests will fail.